arcgis-ios-sdk-samples/Maps/Download preplanned map/README.md

```diff
@@ -0,0 +1,47 @@
+# Download preplanned map
+Take a map offline using a preplanned map area.
+
+![](image1.png)
+
+## Use case
+
+When a map is taken offline, a package containing basemap tiles, feature data, and other resources is created. In the preplanned workflow, the author of the web map has set up the offline packages ahead of time, which enables faster, more resource-efficient downloads compared to an on-demand workflow. Because the resources for the area are packaged once and can be downloaded many times by different users, this approach is more scalable for large organizations. To see the difference for yourself, compare this sample to the Generate an offline map sample.
+
+## How to use the sample
+
+Select a map area to take offline, then use the button to take it offline. Click 'Delete offline areas' to remove any downloaded map areas.
```

Comment:
I think this can be rephrased better to give idea that you just need to select from the available list. I feel that current  wording gives an idea that we have to select a in map like we do with `Generate offline map` sample.

Reply:
Good point. I proposed alternate phrasing via [76573d2b](https://github.com/Esri/arcgis-runtime-samples-ios/pull/758/commits/76573d2b7c82d46d22188e22aa137d5bc6c03f3b). Hopefully you'll find this more to your liking.